### DISCUSSION TITLE
same motif name

### DISCUSSION THREAD

**Discussion by woshiyangsi** [GreenleafLab/ArchR#1678] [General]
Hi,
When I get the motif summary information from obj@peakAnnotation$Motif$motifSummary, I found that there are many same named motifs, such as SIN3A, CTCF...
![image](https://user-images.githubusercontent.com/54662832/195964443-c6dac57a-cadb-4935-ae35-2370a9b65fc3.png)

I'm assuming this is because the components of these same motifs are still somewhat distinct, preventing them from being combined. Could you confirm this, please?

Do these motifs with the same names serve the same functions?

I look forward to your reply.
Siyu.

**Comment by rcorces** [comment]
I'm not sure which annotation you are using but the `ID` column of your spreadsheet indicates to me that the motif PWMs were derived from different experiments in different cell types/cell lines. So the same factor is being profiled in different contexts and I would not be surprised to see that the PWMs resulting from those different experiments are highly similar.

I'd encourage you to read Jeff Vierstra's work on clustering motifs into archetypes which touches on similar themes.
https://www.vierstra.org/resources/motif_clustering
ArchR has these motif archetypes available as an annotation as of `release_1.0.2`

**Reply by woshiyangsi** [reply]
Thank you for your quick reply.
The peaks were annotated by the code:
Merged.proj2 <- addMotifAnnotations(ArchRProj = Merged.proj2,
      motifSet = "encode",
      name = "Motif",species = getGenome(ArchRProj = Merged.proj2 ))

**Comment by woshiyangsi** [comment]
Can the enrichment of CTCF_2 in microglia be considered as microglia-specific enriched TF, if TF of the same name is enriched in different cell types? 
eg. CTCF_1 is enriched in oligodendrocytes, while CTCF_2 is enriched in microglia.

**Reply by rcorces** [reply]
I'm not sure I would believe that interpretation. That would effectively cherry pick one CTCF PWM as being different from another CTCF PWM in a biologically-relevant manner. I'm not sure I believe motif PWMs at that level of information.

**Reply by woshiyangsi** [reply]
I see, thank you very much!
